src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs/crumbTable.jsx

```diff
@@ -12,20 +12,20 @@ const CrumbTable = React.createClass({
   },
 
   renderData() {
-    if (!this.props.kvData) {
+    let {kvData} = this.props;
+    if (!kvData || _.isEmpty(kvData)) {
       return null;
     }
-    return _.chain(this.props.kvData)
-      .map((val, key) => [val, key])
-      .map(([val, key]) => {
-        return (
-          <tr key={key}>
-            <td className="key">{key}</td>
-            <td className="value"><pre>{val + ''}</pre></td>
-          </tr>
-        );
-      })
-      .value();
+    let tuples = _.map(kvData, (val, key) => [val, key]);
```

Comment:
The lodash webpack bundle doesn't support chaining, and lodash's `.map` doesn't play as nice w/ empty objects so I made this more defensive.